### PR TITLE
Impelement get_homogeneous_pages.

### PR DESCRIPTION
**Pull request by mattalbr**
Unit tests forthcoming once we're directionally aligned.

### CODE REVIEW

**Comment by mattalbr** [comment]
#86

File: sqlakeyset/paging.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
```
Comment (acarapetis) [inline sqlakeyset/paging.py:4]: CI fails at flake8 with a bunch of undefined names - looks like you've forgotten some imports?

File: sqlakeyset/paging.py [hunk 2]
```diff
@@ -438,3 +439,87 @@ def get_page(
...
+class PageRequest(Generic[_TP]):
+    """See ``get_page()`` documentation for parameter explanations."""
+    query: Query[_TP]
+    per_page: int = PER_PAGE_DEFAULT
+    after: OptionalKeyset = None
+    before: OptionalKeyset = None
+    page: Optional[Union[MarkerLike, str]] = None
+    
+
+def get_homogeneous_page(queries: list[PageRequest[_TP]]) -> list[Page[Row[_TP]]]:
```
Comment (acarapetis) [inline sqlakeyset/paging.py:455]: I definitely think it should be get_homogeneous_page**s**.

**Review by acarapetis — CHANGES_REQUESTED** [review]

**Comment by acarapetis** [comment]
I definitely think some version of this is worth including, since it's not achievable using our current documented public API; so thanks!

I've added a bunch of minor comments in my review above, should be easy fixes. I do have a couple more significant concerns:

- Since the `Query` API is now considered legacy, I definitely think we want an equivalent `select_homogeneous_pages` API to work with `Select`.
- Unless I'm mistaken, the `_page_identifier` column is going to be hanging around on the rows returned to the user, which is not desirable. We already handle this same issue for added keyset columns in `orm_page_from_rows`, so hopefully we just need to add an additional item to `paging_query.extra_columns` - let me know if you need help with this.

And yeah, bring on the tests. :)

**Review by mattalbr — COMMENTED** [review]
I'm almost there, just still struggling with select_homogeneous_pages. It keeps failing trying to grab the keys. Let me know if you see something obvious, otherwise I'll keep fighting with it.

Otherwise, good for you to take another look.

**Comment by mattalbr** [comment]
Closing in favor of #90 which I finally got working for both ORM and select.

**Pull request closed without merging**